lib/rawQueries.js

```diff
@@ -3,17 +3,17 @@
 const Queries = {
     getPipelineUsageCountForJobTemplatesQuery: (
         tablePrefix = ''
-    ) => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId"
+    ) => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId", "id"
         FROM "${tablePrefix}jobs"
-        WHERE "templateId" IN (:templateIds)
-        GROUP BY "templateId"`,
+        WHERE "templateId" IN (:templateIds) AND "id" IN (:jobIds)
+        GROUP BY "templateId", "id"`,
```

Comment:
Instead of filtering on `jobIds`, why don't we add the filter to ignore archived jobs

```suggestion
        WHERE "templateId" IN (:templateIds) AND "archived" = false
        GROUP BY "templateId"`,
```